src/world_interface/real_world_constants.h

```diff
@@ -99,14 +100,16 @@ constexpr auto motorSerialIDMap = frozen::make_unordered_map<motorid_t, can::dev
 	 {motorid_t::shoulder, DEVICE_SERIAL_MOTOR_SHOULDER},
 	 {motorid_t::elbow, DEVICE_SERIAL_MOTOR_ELBOW},
 	 {motorid_t::forearm, DEVICE_SERIAL_MOTOR_FOREARM},
-	 {motorid_t::wristDiffLeft, DEVICE_SERIAL_MOTOR_WRIST_DIFF_LEFT},
-	 {motorid_t::wristDiffRight, DEVICE_SERIAL_MOTOR_WRIST_DIFF_RIGHT},
+	 {motorid_t::wristDiffLeft, 0xc},
+	 {motorid_t::wristDiffRight, 0x5},
 	 {motorid_t::hand, DEVICE_SERIAL_MOTOR_HAND},
+	 {motorid_t::handActuator, 0x16},
 	 {motorid_t::activeSuspension, DEVICE_SERIAL_LINEAR_ACTUATOR}});
 
 /** @brief A mapping of PID controlled motors to their pid coefficients. */
 constexpr auto motorPIDMap = frozen::make_unordered_map<motorid_t, pidcoef_t>(
-	{{motorid_t::shoulder, {70, 0, 0}},
+	{{motorid_t::shoulder, {200, 0, 0}},
+	 {motorid_t::elbow, {250, 0, 0}},
 	 {motorid_t::frontLeftSwerve, {2, 0, 0}},
 	 {motorid_t::frontRightSwerve, {2, 0, 0}},
 	 {motorid_t::rearLeftSwerve, {2, 0, 0}},
```

Comment:
Major inconsistency with PID on swerve and non-swerve boards, related to PID constant scaling. Non-swerve boards have it correct, swerve boards need to fix how they scale PID power output. This is a firmware issue, and after it's fixed the swerve boards should be retuned. The associated PID constants will be much higher (in the 100s)

Reply:
Notably, PID constants used to be specified in deci-units (i.e. divided by 10) now they will be specified in milli-units (divided by 1000). That is, if you want to set kP=1, you used to send the value 10, now you should use 1000.